ci/test/00_setup_env_win64.sh

```diff
@@ -7,7 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_win64
-export DOCKER_NAME_TAG=ubuntu:20.04  # Check that Focal can cross-compile to win64
+export DOCKER_NAME_TAG=ubuntu:22.04  # Check that Jammy can cross-compile to win64
```

Comment:
I'm confused, isn't Ubuntu 22.04 planned for April this year?

Reply:
Yes, it is. And it is going to be a LTS release.